pages/views/_includes/readme.njk

```diff
@@ -0,0 +1,6 @@
+{% macro readme(componentPath) %}
+  <hr>
+  {% markdown %}
```

Comment:
Please add a div wrapper with some specific class, it will help in the future when we start to add styles.

Reply:
no problem, we can do it in this PR to make it easier later

Reply:
Sorry, just want to propose an alternative name for css class `markdown-container`
Just because it will be abstract styles to fix any type of markdown that we will include into pages.

Reply:
I agree with you